apps/remote_control/lib/lexical/remote_control/analyzer/requires.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Lexical.RemoteControl.Analyzer.Requires do
+  alias Lexical.Ast.Analysis
+  alias Lexical.Ast.Analysis.Require
+  alias Lexical.Ast.Analysis.Scope
+  alias Lexical.Document.Position
+
+  def at(%Analysis{} = analysis, %Position{} = position) do
+    case Analysis.scopes_at(analysis, position) do
+      [%Scope{} = scope | _] ->
+        scope.requires
+        |> Enum.filter(fn %Require{} = require ->
+          require_end = require.range.end
+
+          if require_end.line == position.line do
+            require_end.character <= position.character
+          else
+            require_end.line < position.line
+          end
+        end)
+        |> Enum.uniq_by(& &1.as)
```

Comment:
I notice that `Requires` has this `uniq_by`, but `Uses` doesn't. Just wanted to make sure that's deliberate.

Reply:
it was, my thinking is that you can require multiple things as long as the destination is unique, but I don't see a reason to do this with use.